src/Mgate/SuiviBundle/Controller/EtudeController.php

```diff
@@ -164,11 +164,7 @@ public function addAction(Request $request)
                 $em->persist($etude);
                 $em->flush();
 
-                if ($request->get('ap')) {
-                    return $this->redirect($this->generateUrl('MgateSuivi_ap_rediger', array('id' => $etude->getId())));
-                } else {
-                    return $this->redirect($this->generateUrl('MgateSuivi_etude_voir', array('nom' => $etude->getNom())));
-                }
+                return $this->redirect($this->generateUrl('MgateSuivi_etude_voir', array('nom' => $etude->getNom())));
```

Comment:
Maybe [`return $this->redirectToRoute()` ](https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php#L99)

(Even that `$this->redirect($this->generateUrl(` pattern is used in the application, we should begin to replace it)